bin/node-template/runtime/Cargo.toml

```diff
@@ -16,7 +16,7 @@ codec = { package = "parity-scale-codec", version = "2.0.0", default-features =
 pallet-aura = { version = "3.0.0", default-features = false, path = "../../../frame/aura" }
 pallet-balances = { version = "3.0.0", default-features = false, path = "../../../frame/balances" }
 frame-support = { version = "3.0.0", default-features = false, path = "../../../frame/support" }
-pallet-grandpa = { version = "3.0.0", default-features = false, path = "../../../frame/grandpa" }
+pallet-grandpa = { version = "4.0.0", default-features = false, path = "../../../frame/grandpa" }
```

Comment:
I'm not entirely sure about this. Seems awfully convenient, but on the other hand we don't seem the change these unless we update the substrate version?